src/features/tasks/tasksSlice.js

```diff
@@ -262,6 +272,42 @@ export const tasksSlice = createSlice({
       state.loadingStates.cameraSerialNumber.errors.splice(index, 1);
     },
 
+    // delete camera
+
+    updateDeleteCameraStart: (state) => {
+      let ls = state.loadingStates.deleteCamera;
+      ls.taskId = null;
+      ls.isLoading = true;
+      ls.errors = null;
+    },
+
+    updateDeleteCameraUpdate: (state, { payload }) => {
+      state.loadingStates.deleteCamera.taskId = payload.taskId;
+    },
+
+    updateDeleteCameraSuccess: (state) => {
+      let ls = state.loadingStates.deleteCamera;
+      ls.taskId = null;
+      ls.isLoading = false;
+      ls.errors = null;
+    },
+
+    updateDeleteCameraFailure: (state, { payload }) => {
+      let ls = state.loadingStates.deleteCamera;
+      ls.isLoading = false;
+      ls.errors = [payload.task.output.error];
+    },
+
+    clearDeleteCameraTask: (state) => {
+      state.loadingStates.cameraSerialNumber = initialState.loadingStates.cameraSerialNumber;
```

Comment:
This looks like it may have been copy/pasted from the set of action handlers related to updating a camera's serial number and then not updated? Or is there a reason we want to reset the `cameraSerialNumber` loading state?

Reply:
Bad copy/paste, will fix